autorest/codegen/templates/operations_container_init.py.jinja2

```diff
@@ -7,10 +7,10 @@ from .{{ ("_operations" if code_model.options['combine_operation_files'] else op
 # coding=utf-8
 {{ code_model.options['license_header'] }}
 
-{% if not code_model.options["add_python_3_operation_files"] %}
-{{ operation_group_imports() }}
-{% elif code_model.options["add_python_3_operation_files"] and code_model.options["python_3_only"] %}
+{% if code_model.options["python_3_only"] %}
```

Comment:
@msyyc I'm a little confused why we're removing the `add_python_3_operation_files` checks. Even though that flag's default depends on the `--python3-only` flag, these are 2 separate flags, and can be configured separately

Reply:
I think for this PR, really the only line you should have to change in the actual codegen code is to have the default of `add_python_3_operation_files` be `python3_only or version_tolerant`

Reply:
I try to make the logic more simple. If `python3-only` is `True`, `add_python_3_operation_files` must be `True`. So there is no need to judge it when `python3-only` is `True`.

Reply:
@msyyc if `python3-only` is `True`, that does not mean that `add-python3-operation-files` is `True`, it just means that `add-python3-operation-files` defaults to `True`. If users pass in `--python3-only --add-python3-operation-files=False`, `python3-only` would be `True` and `add-python3-operation-files` would be `False`.

You could ask why they would do that, but we have a general policy of not disallowing things, so if users really want, say python3 models files, but no python3 sync operation files, that's up to them

Reply:
@iscai-msft I have to admit that it is a little confusing. But you are right. We need to think about this scenario.
In view of `aio` folder only have `_operations.py` so I keep your logic and add additional judgement for `async_mode`.

https://github.com/Azure/autorest.python/blob/a51ae57a2074052634e61c8e2644e64bfd2a8792/autorest/codegen/templates/operations_container_init.py.jinja2#L10